pkg/knative/commands/deployer_create.go

```diff
@@ -249,7 +249,9 @@ Create a Knative deployer.
 
 Build references are resolved within the same namespace as the deployer. As the
 build produces new images, the image will roll out automatically. Image based
-deployers must be updated manually to roll out new images.
+deployers must be updated manually to roll out new images. Rolling out an image
+means creating a Knative Configuration with a pod referencing the image and a
```

Comment:
should "pod" be "pod spec" or "PodSpec"?

Reply:
I think there was a move to change from a Pod to PodSpec, but either way, I think we can keep this text informal and just say "pod".